distributor-api-v1/operations.md

```diff
@@ -760,34 +760,45 @@ Gives a pricing information for the given configuration.
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `HotelId` | string | required | Unique identifier of hotel |
-| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization) |
-| `Session` | number | required | Session number |
-| `LanguageCode` | string | required | Language code |
-| `CultureCode` | string | required | Culture Code |
+| `HotelId` | string | required | Unique identifier of hotel. |
+| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
+| `Session` | number | required | Session number. |
+| `LanguageCode` | string | required | Language code. |
+| `CultureCode` | string | required | Culture Code. |
 
-### Response   <a id="response-5"></a>
+### Response  <a id="response-getpaymentconfiguration"></a>
 
 ```javascript
 {
-    {
-        "PaymentGatewayType": "Adyen",
-        "IsMerchant": false,
+    "PaymentGateway": {
+        "PaymentGatewayType": "PciProxy",
+        "PaymentCardStorageType": "PciProxy",
+        "IsMerchant": true,
         "SupportedCreditCardTypes": [
             "MasterCard",
             "Visa"
         ],
-        "PublicKey": "..."
+        "PublicKey": "1100016614",
+        "DefaultCurrencyCode": "EUR"
+    },
+    "SurchargeConfiguration": {
+        "SurchargeServiceId": null,
+        "SurchargeFees": {}
     }
 }
 ```
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| PaymentGatewayType | string | required | Type of the payment gateway \(`Adyen`, `Stripe` or `PciProxy`\) |
-| IsMerchant | boolean | required | Whether the gateway is processed through Mews Merchant or not |
-| SupportedCreditCardTypes | array of strings | required | The list of supported credit cards, should be used to enhance UX |
-| PublicKey | string | required | A key used by the payment gateway's client side library for authentication |
+| `PaymentGateway` | [PaymentGateway](operations.md#payment-gateway) | required | Object that describes Payment gateway of the enterprise. |
+| `SurchargeConfiguration` | [SurchargeConfiguration](operations.md#surcharge-configuration) | required | Object that describes surcharge configuration that enterprise uses. |
+
+#### Surcharge Configuration <a id="surcharge-configuration"></a>
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `SurchargeServiceId` | string | optional | Unique identifier of surcharge service, if enterprise uses surcharge feature this Id will be provided. |
+| `SurchargeFees` | dictionary of string/decimal | optional | Dictionary of surcharge fees where string is `PatmentCardType`(such as Visa, MasterCard, Amex ...) and decimal is amount of the surcharge fee itself. |
```

Comment:
I don't think we describe it as dictionary instead it should be maybe kind of similar to for example 
![image](https://user-images.githubusercontent.com/6526378/74835315-54cb7b80-531d-11ea-8644-a383c4fc0689.png)

link with a special section where this is described.

And also it's better to call in object, since that's what FE has? LocalizedText is described like that.